cl/recap/models.py

```diff
@@ -211,6 +212,53 @@ def file_contents(self):
     def print_file_contents(self):
         print(self.file_contents)
 
+class EmailProcessingQueue(AbstractDateTimeModel):
+    uploader = models.ForeignKey(
+        User,
+        help_text="The user that uploaded the item to RECAP.",
+        related_name="recap_email_processing_queue",
+        on_delete=models.CASCADE,
+        null=True,
```

Comment:
You shouldn't need a null here, I don't think. We'll always know the uploader when the API request is received.

Reply:
This is going to be a standard recap user, correct?

Reply:
Yes, generally. I guess what we should do is create a new user, `recap.email` and then use that user's credentials. To do that, we could just use the approach here: https://github.com/freelawproject/courtlistener/blob/5b5f90470f550a17998595f70272b7489904299a/cl/users/migrations/0004_make_recap_user.py. We could use the `recap` user created in that file, but it's easy enough to do that it seems like having a separate user is probably a good idea to create a security boundary between the services.